org.eclipse.xtext.testing/xtend-gen/org/eclipse/xtext/testing/AbstractLanguageServerTest.java

```diff
@@ -189,58 +185,22 @@ public synchronized <U extends Object, V extends Object> CompletableFuture<V> ru
     }
   }
 
-  @Accessors
-  public static class TestCodeLensConfiguration extends TextDocumentPositionConfiguration {
+  /* @Accessors
+   */public static class TestCodeLensConfiguration extends TextDocumentPositionConfiguration {
```

Comment:
Something's off with the generated code.

Reply:
i regenerated the file